### DISCUSSION TITLE
i got an error when executing phalcon on xampp "Wrong number of parameters"

### DISCUSSION THREAD

**Discussion by Tediace** [phalcon/cphalcon#16025] [General]
Wrong number of parameters
#0 C:\xampp1\htdocs\crud_by_phalcon\app\config\router.php(7): Phalcon\Mvc\Router->handle()
#1 C:\xampp1\htdocs\crud_by_phalcon\public\index.php(20): include('C:\\xampp1\\htdoc...')
#2 {main}

'''<?php

$router = $di->getRouter();

// Define your routes here

$router->handle();

**Comment by niden** [comment]
`handle()` requires a URI string. Try this

```php
$router->handle($_SERVER['REQUEST_URI']);
```

**Reply by Tediace** [reply]
thank u, sir
